tests/e2e-http-server/http-custom-certs/50-verify-metrics.yaml

```diff
@@ -27,8 +27,11 @@ data:
     set -o errexit
 
     # metrics can change only validate a portion of it
-    curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/v1/metrics | tee /tmp/curl.out
-    grep -cq 'vertica_build_info' /tmp/curl.out
+    for endpoint in metrics v1/metrics
+    do
+      curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/$endpoint | tee /tmp/curl.out
+      grep -cq 'vertica_build_info' /tmp/curl.out && break
```

Comment:
We have `set -o errexit` set, so when the endpoint is v1/metrics, won't this fail during the first iteration?

Reply:
No I think the exception is when the command is part of an AND/OR-list. I will reverse the endpoints order just to be sure.

Reply:
Tricky. It's because we don't have `set -o pipefail`. The `curl` command that pipes to `tee` will actually succeed because the `tee` command is a success. Maybe add a comment here.

Reply:
I will!